gitlab/__init__.py

```diff
@@ -20,6 +20,7 @@
 from typing import Any
 
 import gitlab.config  # noqa: F401
+from gitlab import utils as _utils
```

Comment:
Not sure I fully understand the context here for `_utils` compared to how it's imported in the rest of the library. If it's needed, maybe we just go with `_utils.py` then to avoid `import thing as another_thing` and be consistent? Would still be very pep8-ish:

> Even with `__all__` set appropriately, internal interfaces (packages, modules, classes, functions, attributes or other names) should still be prefixed with a single leading underscore.

Reply:
I did that as I didn't want to possibly interfere with usage of `gitlab.utils`. Not positive this is necessary but was doing it to be safe.